configuration/metadatatermmappings/metadataMappings.csv

```diff
@@ -14,4 +14,5 @@ bc976a45-1d9e-4193-a541-a689ba38cf69,,org.openmrs.module.emrapi,emr.transferWith
 6c63c917-6b57-458c-8f36-f460ca70f67d,,org.openmrs.module.emrapi,emr.transferWithinHospitalForm,org.openmrs.Form,d068bc80-fb95-11e2-b778-0800200c9a66
 fdcc4b43-1e55-4ba4-8392-c1860de885b9,,org.openmrs.module.emrapi,emr.motherChildRelationshipType,org.openmrs.RelationshipType,${relationshipType.motherToChild.uuid}
 a1b2c3d4-e5f6-7a8b-9c0d-1e2f3a4b5c6d,,org.openmrs.module.emrapi,emr.inpatientNoteEncounterType,org.openmrs.EncounterType,${encounterType.INPATIENT_NOTE.uuid}
-b1c2d3e4-f5a6-7b8c-9d0e-1f2a3b4c5d6e,,org.openmrs.module.emrapi,emr.transferRequestEncounterType,org.openmrs.EncounterType,${encounterType.TRANSFER_REQUEST.uuid}
\ No newline at end of file
+b1c2d3e4-f5a6-7b8c-9d0e-1f2a3b4c5d6e,,org.openmrs.module.emrapi,emr.transferRequestEncounterType,org.openmrs.EncounterType,${encounterType.TRANSFER_REQUEST.uuid}
+d039435f-233a-4d0b-a35a-75450b2e65b9,,org.openmrs.module.emrapi,emr.bedAssignmentEncounterType,org.openmrs.EncounterType,${encounterType.BED_ASSIGNMENT.uuid}
```

Comment:
You actually will still need the line to here to set up the metadata mapping for Cancel ADT, it just should point to the existing CANCEL_ADMISSION

Reply:
yeah, just realized that. Sorry I jumped the gun. 

I also need an entry in messages_en.properties right?

Reply:
Thanks @chibongho ... there's probably already an entry in messages.properties for this encounter type, so you probably don't need it in, unless we want to change the display name to Cancel ADT, which might make sense?